### PR TITLE
LPS-147701 | Test Fix Upgrade should not run on Portal CE

### PR DESCRIPTION
**Pull request by Hijky**
**Background:**
Test Fix Upgrade should not run on Portal CE

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-147701

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsUpgrade.testcase [hunk 1]
```diff
@@ -21,6 +21,7 @@ definition {
 	@priority = "4"
 	test CanViewRemoteAppCustomElementPortletAfterUpgrade7413 {
 		property portal.acceptance = "true";
+		property test.run.environment = "EE";
 		property data.archive.type = "data-archive-remote-app";
 		property portal.version = "7.4.13";
```
Comment (zhangyan0701) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsUpgrade.testcase:26]: Hi @Hijky 

Could you reorder the properties in alphabetical ascending order? And also update test CanViewRemoteAppFieldsAfterUpgrade73101.

Thanks!

**Review by zhangyan0701 — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsUpgrade.testcase (hunk 1 above)
Comment (Hijky) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsUpgrade.testcase:26]: ok, I will edit and re-pull it today

**Comment by Hijky** [comment]
ci:test:relevant

**Comment by Hijky** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
Hi @Hijky 

Please update the commit message to start with verb, such as `Add property to only run on portal EE` or `Fix test to not run on portal CE`.

You can run ci:forward after the update.

**Comment by Hijky** [comment]
ci:forward

**Review by zhangyan0701 — APPROVED** [review]

**Comment by Hijky** [comment]
I will resend it

**Pull request closed without merging**